source/fuzz/fact_manager/data_synonym_and_id_equation_facts.cpp

```diff
@@ -897,6 +897,11 @@ DataSynonymAndIdEquationFacts::GetIdsForWhichSynonymsAreKnown() const {
   return result;
 }
 
+std::vector<const protobufs::DataDescriptor*>
+DataSynonymAndIdEquationFacts::GetAllKnownSynonyms() const {
+  return synonymous_.GetAllKnownValues();
+}
```

Comment:
In some more recent changes I have done some work to ensure that data descriptors whose objects no longer exist in the module don't get returned.  I think it would be good to do that here too.

Reply:
Fixed in 4514ef2.